### ISSUE TITLE
Downloading mod 111111111 ... Mod 111111111 was not successfully downloaded

### ISSUE THREAD

**Issue by NLBlackEagle** [FezVrasta/ark-server-tools#558]
Hello everyone,

When trying to perform the next command in linux:
`arkmanager update --warn --saveworld --update-mods`

I get the following message narrated:

Running command 'update' for instance 'main'
Downloading mod 111111111 ... Mod 111111111 was not successfully downloaded

While the mod is not listed under ark_GameModIds, here's the content:
`ark_GameModIds="510639902,519998112,489051927,651584871,658112994,618916953,554678442,679529026,670764308,539464369,503911420,612504284,520879363,497432858,543721929,600015460,655261420,679500536,695990475,663332507,682394267"`

This can be resolved by using:
`arkmanager uninstallmod 111111111` but it is not preferable.

Thanks for helping me out/fixing this issue in advance!

**Comment by klightspeed** [comment]
This should have already been fixed by the latest update to arkmanager.

**Comment by NLBlackEagle** [comment]
Oh my bad, I totally forgot to update!

**Comment by Torath0071** [comment]
Hi, to solve this problem, i've removed the folder 111... from Mods folder.
arkmanager do not try to update this mod.

By the way, arkmanager update all mods in Mods folder even if is not enable in cfg file

**Comment by klightspeed** [comment]
Commit 8a59a8c7 should have made arkmanager ignore mod 111111111

**Comment by arkfrance** [comment]
so are you updating your P+ with the arkmanager update --update-mods ?

**Comment by klightspeed** [comment]
Primitive+ cannot be updated using the `--update-mods` mechanism because it is not on the Steam workshop.  Primitive+ comes with the base install of the ARK server (Steam AppId 376030).

**Comment by arkfrance** [comment]
Good to confirm thanks

**Comment by arkfrance** [comment]
Please, close your ticket if you think it's resolved :)

**Comment by boistordu** [comment]
since the guy who run the docker image for ark with arkmanager is not answering, 
can you tell me how to update manually the arkmanager? because I'have this:
```
root@84e865219a21:/ark# arkmanager --version
Version: 1.5
Channel: 
root@84e865219a21:/ark# arkmanager upgrade-tools
arkmanager v1.5: Checking for updates...
A new version was found! Do you want to upgrade ARK Server Tools to v404: Not Found?y
bash: line 1: 404:: command not found
root@84e865219a21:/ark# 
```

**Comment by klightspeed** [comment]
That looks like arkstChannel is not set.  Please add `arkstChannel=master` to the top of your `arkmanager.cfg`, and try again.

**Comment by boistordu** [comment]
still no luck :+1: 
```

root@186efac438c9:/ark# arkmanager --version
Version: 1.5
Channel: master
root@186efac438c9:/ark# arkmanager upgrade-tools
arkmanager v1.5: Checking for updates...
A new version was found! Do you want to upgrade ARK Server Tools to v404: Not Found?y
bash: line 1: 404:: command not found
root@186efac438c9:/ark# 
```

anyway it's normally up to date because at each creation of dockers' container,  it does `RUN  git clone https://github.com/FezVrasta/ark-server-tools.git /home/steam/ark-server-tools`

but it also means that the download mod111111 error is not corrected. Or it doesn t download the last commit. 
So how can I check the commit?

**Comment by boistordu** [comment]
```
root@186efac438c9:/home/steam/ark-server-tools# git checkout master
Previous HEAD position was 46f8c16... Merge pull request #399 from FezVrasta/1.5-dev
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.
```

**Comment by klightspeed** [comment]
```
git status
```

**Comment by boistordu** [comment]
that's not the commit friend :) 
```
root@186efac438c9:/home/steam/ark-server-tools# git status
On branch master
Your branch is up-to-date with 'origin/master'.

nothing to commit, working directory clean
```

**Comment by boistordu** [comment]
mmmmh so in checkout it was the commit ok ...

**Comment by boistordu** [comment]
ok I think I found the problem the GIT_TAG in dockerfile. the git was fixed on 1.5

**Comment by boistordu** [comment]
I don't have the downloading 1111 at installation anymore so I guess it's okey
